src/test/java/com/github/dockerjava/core/command/StopContainerCmdImplTest.java

```diff
@@ -63,7 +67,13 @@ public void testStopContainer() throws DockerException {
         LOG.info("Container Inspect: {}", inspectContainerResponse.toString());
 
         assertThat(inspectContainerResponse.getState().getRunning(), is(equalTo(false)));
-        assertThat(inspectContainerResponse.getState().getExitCode(), not(equalTo(0)));
+
+        final Integer exitCode = inspectContainerResponse.getState().getExitCode();
+        if (apiVersion.equals(VERSION_1_22)) {
+            assertThat(exitCode, is(0)); // FIXME: 2/26/16
```

Comment:
Checked locally with the same result. For 1.22 exit code is 0 and for 1.21 exit code is 137...